litestar/middleware/compression/middleware.py

```diff
@@ -82,10 +83,22 @@ async def __call__(self, scope: Scope, receive: Receive, send: Send) -> None:
 
         await self.app(scope, receive, send)
 
+    def get_facade_cls(
+        self,
+        compression_encoding: Literal[CompressionEncoding.BROTLI, CompressionEncoding.GZIP, CompressionEncoding.ZSTD]
+        | str,
+    ) -> type[CompressionFacade]:
+        if compression_encoding == CompressionEncoding.GZIP:
+            return GzipCompression
+        if compression_encoding == CompressionEncoding.ZSTD:
```

Comment:
```suggestion
        elif compression_encoding == CompressionEncoding.ZSTD:
```

`elif` is better when working with `Literal` values. 

Is that correct that we don't use `assert_never()` here?

Reply:
I'd say no, to allow arbitrary user-supplied encodings